test/common/schemaform/FormApp.unit.spec.jsx

```diff
@@ -83,4 +84,104 @@ describe('Schemaform <FormApp>', () => {
 
     expect(tree.everySubTree('LoadingIndicator')).not.to.be.empty;
   });
+  it('should route when prefill complete', () => {
+    const formConfig = {
+      title: 'Testing'
+    };
+    const currentLocation = {
+      pathname: 'test'
+    };
+    const routes = [{
+      pageList: [{}, {
+        path: 'test-path'
+      }]
+    }];
+    const router = {
+      push: sinon.spy()
+    };
+
+    const tree = SkinDeep.shallowRender(
+      <FormApp
+          formConfig={formConfig}
+          currentLocation={currentLocation}
+          loadedStatus={LOAD_STATUSES.pending}
+          prefillStatus={PREFILL_STATUSES.pending}
+          isLoggedIn
+          updateLogInUrl={() => {}}>
+        <div className="child"/>
+      </FormApp>
+    );
+
+    tree.getMountedInstance().componentWillReceiveProps({
+      prefillStatus: PREFILL_STATUSES.complete,
+      router,
+      routes
+    });
+
+    expect(router.push.called).to.be.true;
```

Comment:
Should we check to see what it was called with? Just in case it's called with `/error`, for example.